.editorconfig

```diff
@@ -148,6 +148,21 @@ dotnet_diagnostic.RS2008.severity = none
 dotnet_diagnostic.IDE0073.severity = warning
 file_header_template = Licensed to the .NET Foundation under one or more agreements.\nThe .NET Foundation licenses this file to you under the MIT license.\nSee the LICENSE file in the project root for more information.
 
+# IDE0035: Remove unreachable code
+dotnet_diagnostic.IDE0035.severity = warning
+
+# IDE0036: Order modifiers
+dotnet_diagnostic.IDE0036.severity = warning
+
+# IDE0040: Add accessibility modifiers
+dotnet_diagnostic.IDE0040.severity = warning
+
+# IDE0043: Format string contains invalid placeholder
+dotnet_diagnostic.IDE0043.severity = warning
+
+# IDE0044: Make field readonly
+dotnet_diagnostic.IDE0044.severity = warning
```

Comment:
Current PR state:

- IDE0035:
  - C#: No violations were detected.
  - VB: No violations were detected.
- IDE0036:
  - C#: Violations were detected and fixed.
  - VB: Violations were detected, but still not addressed.
- IDE0040:
  - C#: Violations were detected but still not addressed.
  - VB: Violations were detected, but still not addressed.
- IDE0043:
  - C#: No violations were detected.
  - VB: No violations were detected.
- IDE0044:
  - C#: Violations were detected and fixed.
  - VB: Violations were detected, but still not addressed.

The PR now edits 138 C# file.
I'm unsure of how large changes you can accept.
I'll stop fixing more violations.

To keep the PR small as possible, my suggestion would be one of the following:

1. Exclude IDE0040 from this PR, and keep it for a later PR.
1. Exclude IDE0040, and also exclude VB from IDE0036 and IDE0044.

But it's up to you on how to continue with this.